Websites/webkit.org/demos/webgpu/scripts/hello-triangle.js

```diff
@@ -30,17 +30,17 @@ async function helloTriangle() {
 /*
```

Comment:
did you want to uncomment the wgsl source and remove (or comment out) the metal source from this file?

Reply:
I did that locally, but I was waiting until we could display the colors to replace the metal source